src/main/java/Window/Controller.java

```diff
@@ -1,22 +1,75 @@
-package Window;
+package window;
 
-import javafx.application.Application;
 import javafx.application.Platform;
 import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
 import javafx.scene.control.TextArea;
 import javafx.scene.input.MouseEvent;
-import javafx.scene.layout.Pane;
+import javafx.scene.control.Button;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.stage.Stage;
+import javafx.scene.Scene;
+import javafx.scene.control.MenuBar;
+import javafx.scene.control.MenuItem;
 
+import java.io.IOException;
+import java.io.File;
+import javafx.stage.FileChooser;
 import java.text.SimpleDateFormat;
 
 /**
  * Created by 101010 on 8-5-2017.
  */
 public class Controller {
+    @FXML private Button browse;
+
+    @FXML private MenuBar menu;
+
+    @FXML private MenuItem openFile;
+
     @FXML private TextArea console;
 
+    @FXML private void open(final ActionEvent event) throws IOException {
+        Stage stage = (Stage) menu.getScene().getWindow();
+        Parent root;
+        if (event.getSource() == openFile) {
+            final FileChooser fileChooser = new FileChooser();
+            FileChooser.ExtensionFilter extFilter = new FileChooser.ExtensionFilter("GFA files (*.gfa)", "*.gfa");
+            fileChooser.getExtensionFilters().add(extFilter);
+            final Button openButton = new Button("Open");
+            File file = fileChooser.showOpenDialog(stage);
+            if (file != null) {
+                //TODO open new file
```

Comment:
Here the file should be parsed and visualized, but that's not working yet, so not included here. Will update the TODO to make that clearer.